org.eclipse.scanning.points/src/org/eclipse/scanning/points/RepeatedPointIterator.java

```diff
@@ -81,4 +67,21 @@ public int getRank() {
 		return 1;
 	}
 
+	// TODO: 2017-10-09 Matt D: These two fields and the get and set methods for countSleeps exist
+	// only to test/ the number of times the iterator is iterated over during a scan in ScanProcessTest
+	// We should find another way to do this
+	private static boolean countSleeps;
+	private static int     sleepCount;
+
+	public static void _setCountSleeps(boolean count) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)